src/System.Diagnostics.Debug/tests/DebugTestsNoListeners.cs

```diff
@@ -0,0 +1,210 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection;
+using Xunit;
+
+namespace System.Diagnostics.Tests
+{
+    // These tests test the static Debug class. They cannot be run in parallel
+    // DebugTestsNoListeners: when run alone, tests Debug behavior before Debug is set with Trace Listeners.
+    [Collection("System.Diagnostics.Debug")]
+    public class DebugTestsNoListeners : DebugTests
```

Comment:
`DebugTestsNoListeners` tests Debug APIs prior to ever making any call to any `Trace` APIs.